emission/net/ext_service/otp/planner.json

```diff
@@ -1,3 +1,3 @@
 {
-	"open_trip_planner_instance_address" : "http://54.162.140.55:8080"
+	"open_trip_planner_instance_address" : "http://18.209.111.117"
```

Comment:
did you add instructions about how to launch OTP using your docker instance + load data?